cdk/lib/support-workers.ts

```diff
@@ -178,6 +169,58 @@ export class SupportWorkers extends GuStack {
         });
     };
 
+    const createTypescriptLambda = (
+      lambdaName: string,
+      additionalPolicies: PolicyStatement[] = []
+    ) => {
+      const lambdaId = `${lambdaName}Lambda`;
+      const lambdaTSFile = lambdaId.charAt(0).toLowerCase() + lambdaId.slice(1);
+      const lambda = new GuLambdaFunction(this, lambdaId, {
+        ...lambdaDefaultConfig,
+        app: "support-workers-typescript",
+        fileName: `support-workers.zip`,
+        runtime: Runtime.NODEJS_20_X,
+        handler: `lambdas/${lambdaTSFile}.handler`,
+        functionName: `${this.stack}-${lambdaName}Lambda-${this.stage}`,
+        initialPolicy: [
+          s3Policy,
+          cloudWatchLoggingPolicy,
+          parameterStorePolicy,
+          ...additionalPolicies,
+        ],
+      });
+      this.overrideLogicalId(lambda, {
```

Comment:
Is this needed? Does the lambda already exist?

Reply:
It's not for the dummy lambda but it will be once we start adding the real ones in.